descartes_light/core/include/descartes_light/solvers/ladder_graph/impl/ladder_graph_solver.hpp

```diff
@@ -177,7 +177,7 @@ BuildStatus LadderGraphSolver<FloatType>::buildImpl(
       // capacity is reached so this could be huge when solving large ladder graph problems.
       // So shrink the capacity to fit
       // @todo Should max possible size be reserved first
-      from_node.edges.shrink_to_fit();
+      // from_node.edges.shrink_to_fit();
```

Comment:
Uncomment?

Reply:
This should be resolve with recent changes to further improve the number of allocations